contracts/v0.1/scripts/sol-error-mappings/errorMappings.json

```diff
@@ -0,0 +1,71 @@
+{
+  "TooManyConsumers()": "05a48e0fc42b9cfeed3601aff0cfc0d560d245d28de6c384623d71dc535c2272",
+  "MustBeRequestedOwner(address)": "d084e975a962ac435d13cd622ad247f5a0482af4cce640e1af8dc9e5e88a9c36",
```

Comment:
it's probably gonna have different hash if address is replaced with specific value, did you also check the case? would it be still recognizable with this hash value?

Reply:
just a suggestion, but how about making a yarn command taking hash and return decoded custom error using related library?
i found this kind of answer in stackoverflow
https://ethereum.stackexchange.com/a/153295

Reply:
regarding, hash correctness, i found the following piece:

In Solidity, the error signature is generated from its name and the types of its parameters. The parameter names are not included in the signature. This is similar to how function signatures are generated in Solidity.

The signature is then hashed using the Keccak256 algorithm to produce a 4-byte identifier. This identifier is what's used in the bytecode to represent the error.

so as long as the error definitions dont change the encoding/hash won't change. And if the error def changes, we can always generate updated hashes by running this script. Pls let me know if I misunderstood your point.

Reply:
awh I see, think I misunderstood how hashes were generated. thank you for the explanation.
still, it'd be nice to have if we can parse the data together if there's a parameter in there error  😄

Reply:
yes that'd be great! I'm currently testing out the library that you suggested, `ethers-decode-error`, from vrf-consumer script but I keep running into errors when I try to initialize the decoder (it's failing to parse the passed abi) or cannot parse the error if no abi is passed. Maybe we can take a quick look together on Monday :)